### DISCUSSION TITLE
coKreat local installation

### DISCUSSION THREAD

**Discussion by Kovendan03** [Sunbird-coKreat/Community#27] [General Discussions]
Hi Team,

I followed the documentation for installing the cokreat locally in ubuntu - https://cokreat.sunbird.org/use/installation-guide/ubuntu-or-mac-installation, it throws error saying "gyp: binding.gyp not found (cwd: /root/creation-portal/src/app) while trying to load binding.gyp" not found.

Is this file needs to be part of the src directory.

**Comment by arun1205** [comment]
At which Stage of installation u got this error @Kovendan03

**Comment by Kovendan03** [comment]
@arun1205, Thanks @arun1205 for your reply,

when i run the "npm i" in  \creation-portal\src\app, i am getting this error.

Change directory to ..\creation-portal\src\app
set NODE_OPTIONS=--max_old_space_size=4096 
npm i

**Reply by arun1205** [reply]
@Kovendan03 can u join the link @ 4pm today to further discuss this meet.google.com/aoq-hian-icu

**Reply by Kovendan03** [reply]
@arun1205 , Will connect with you, to resolve the issue.

**Reply by arun1205** [reply]
@Kovendan03 you can provide your emailID  or you cam mail me @ arun.kumar@tarento.com
